Lib/fontmake/__main__.py

```diff
@@ -346,8 +346,8 @@ def main(args=None):
     )
     compatibilityGroup.add_argument(
         "--no-check-compatibility",
-        action="store_true",
-        dest="disable_compatibility_check",
+        action="store_false",
+        dest="check_compatibility",
```

Comment:
I think you also need to add default=None to both options, otherwise store_false/store_true will set the default to the opposite bool value, no?